packages/angular-sdk-components/src/lib/_services/theme.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { Theme } from '../_types/Theme.interface';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ThemeService {
+  private readonly themes: Theme[] = [
+    {
+      id: 'default-theme',
+      primary: '#8F00FF',
+      displayName: 'Default Theme'
+    },
+    { id: 'dark-theme', primary: '#FF00FF', displayName: 'Dark Theme' },
+    { id: 'red-theme', primary: '#FF0000', displayName: 'Red Theme' },
+    { id: 'azure-theme', primary: '#007FFF', displayName: 'Azure Theme' },
+    { id: 'green-theme', primary: '#008000', displayName: 'Green Theme' }
+  ];
+
+  getDefaultTheme(): string {
```

Comment:
Shoudn't we call this method `getCurrentTheme` or `getActiveTheme` since it's going to return the active/current theme?